src/main.cpp

```diff
@@ -4,15 +4,14 @@ int main(int argc, char** argv)
 {
 	if (argc != 2) {
 		std::cerr << "error: arguments invalid\nexpected: ";
-		std::cerr << argv[0] << " <config file>\n";
+		std::cerr << argv << " <config file>\n";
```

Comment:
Interesting! This was done because of clang-tidy (doesn't like the array access). 

In glibc there is also program_invocation_name or program_invocation_short_name: [LINK](https://man7.org/linux/man-pages/man3/program_invocation_name.3.html), which is defined in \<cerrno\>

With it we wouldn't need to deactivate the linter and be more readable. So it would be 
`std::cerr << program_invocation_name << " <config file>\n";`
which would give us "./webserv \<config file\>"

Reply:
Works for me, if it does work!